qngng/qngng.py

```diff
@@ -155,6 +164,10 @@ def _parse_args():
                         help='Category name')
     parser.add_argument('--double-surname', '-d', action='store_true',
                         help='Create a double-barrelled surname (only available for the `std` category)')
+    parser.add_argument('--middle-name', '-M', action='store_true',
```

Comment:
--middle-initial

Reply:
You want to replace --middle-name with --middle-initial? Or this is an old comment?

Reply:
Old I guess.

It's okay to have both `--middle-name` and `--middle-initial`, as long as you make sure that we pass only one of them.

Reply:
Yes perfect, on it!